pkgs/development/python-modules/pydeck/default.nix

```
@@ -0,0 +1,25 @@
+{ lib, buildPythonPackage, fetchPypi, ipykernel, ipywidgets }:
+
+buildPythonPackage rec {
+  pname = "pydeck";
+  version = "0.7.0";
+
```

```suggestion
  version = "0.7.0";

  disabled = pythonOlder "3.7";
```

see https://github.com/visgl/deck.gl/blob/master/bindings/pydeck/setup.py#L196

---

done.